recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
   run:
     - python >={{ python_min }}
     - boto3 >=1.10.44,<2.0
+    - sagemaker-jupyterlab-extension-common >=0.2.1,<1
```

Comment:
are we good with any value of `sagemaker-jupyterlab-extension-common` above 0.2.1